Formula/jupyterlab.rb

```diff
@@ -375,26 +383,43 @@ class Jupyterlab < Formula
   end
 
   resource "websocket-client" do
-    url "https://files.pythonhosted.org/packages/8d/12/cd10d050f7714ccc675b486cdcbbaed54c782a5b77da2bb82e5c7b31fb40/websocket-client-1.3.1.tar.gz"
-    sha256 "6278a75065395418283f887de7c3beafb3aa68dada5cacbe4b214e8d26da499b"
+    url "https://files.pythonhosted.org/packages/0e/e7/e705ead133d21de4be752af4b3a0cb1f02514ff45bf165b3955c1ce22077/websocket-client-1.3.3.tar.gz"
+    sha256 "d58c5f284d6a9bf8379dab423259fe8f85b70d5fa5d2916d5791a84594b122b1"
   end
 
   def install
-    venv = virtualenv_create(libexec, Formula["python@3.9"].opt_bin/"python3")
+    venv = virtualenv_create(libexec, "python3")
```

Comment:
Interestingly, when `python@3.9` and `python@3.10` are in build environment, this picks 3.9 on Linux. Logs in CI show site-packages has python3.9 in path:
```
/home/linuxbrew/.linuxbrew/Cellar/jupyterlab/3.4.3/libexec/lib/python3.9/site-packages/jupyterlab_server/translation_utils.py
```